lib/src/Uncertainty/Distribution/WeibullMinFactory.cxx

```diff
@@ -89,6 +90,17 @@ WeibullMin WeibullMinFactory::buildAsWeibullMin(const Sample & sample) const
   }
 }
 
+WeibullMin WeibullMinFactory::buildMethodOfLikelihoodMaximization(const Sample & sample) const
+{
+  const MaximumLikelihoodFactory factory(buildMethodOfMoments(sample));
+  return buildAsWeibullMin(factory.build(sample).getParameter());
+}
+
+WeibullMin WeibullMinFactory::buildAsWeibullMin(const Sample & sample) const
+{
+  return buildMethodOfLikelihoodMaximization(sample);
```

Comment:
Please add a Log info about the used method

Reply:
ok